Mathlib/Computability/DFA.lean

```diff
@@ -91,11 +102,19 @@ theorem evalFrom_of_append (start : σ) (x y : List α) :
   x.foldl_append _ _ y
 #align DFA.eval_from_of_append DFA.evalFrom_of_append
 
+/--
+`M.acceptsFrom start` is the language of `x` such that `M.evalFrom start x` is an accept state.
+-/
+def acceptsFrom (start : σ) : Language α := {x | M.evalFrom start x ∈ M.accept}
```

Comment:
I would call the argument just `(s : σ)`. The current name has a potential for confusion with `M.start`.

Reply:
I agree that I currently read it as a declaration name, not a variable name.

Reply:
That's what I meant, thanks for putting it into appropriate words!

Reply:
Thanks, renamed to `s`.